examples/local/201_customer_tablets.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Copyright 2020 The Vitess Authors.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# this script creates the tablets and initializes them for vertical
+# resharding it also splits the vschema between the two keyspaces
+# old (commerce) and new (customer)
+
+source ./env.sh
+
+for i in 200 201 202; do
+ CELL=zone1 TABLET_UID=$i ./scripts/mysqlctl-up.sh
+ CELL=zone1 KEYSPACE=customer TABLET_UID=$i ./scripts/vttablet-up.sh
+done
+
+vtctlclient -server localhost:15999 InitShardMaster -force customer/0 zone1-200
+vtctlclient -server localhost:15999 ApplyVSchema -vschema_file vschema_commerce_vsplit.json commerce
+vtctlclient -server localhost:15999 ApplyVSchema -vschema_file vschema_customer_vsplit.json customer
```

Comment:
We should try and run this as either an ALTER with the mysql protocol, or use `-vschema '{"tables": {"product": {}}}'`.

This will reduce the level of indirection in the docs and make it easier to follow what is happening.

Reply:
Have used -vschema